drivers/hud_twenty_twenty_two_to_twenty_twenty_four/app/models/hud_twenty_twenty_two_to_twenty_twenty_four/enrollment/add_rental_subsidy_type.rb

```diff
@@ -0,0 +1,29 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HudTwentyTwentyTwoToTwentyTwentyFour::Enrollment
+  class AddRentalSubsidyType
+    SUBSIDY_TYPES = {
+      28 => 428,
+      19 => 419,
+      31 => 431,
+      34 => 433,
+      20 => 434,
+      3 => 420,
+    }.freeze
+    def process(row)
+      row['RentalSubsidyType'] = nil
+
+      situation = row['LivingSituation'].to_i
```

Comment:
is `LivingSituation` the right thing to check here?

Reply:
Yes -- the mapping document is looking for 3.917A.1 or 3.917B.1 both of which are stored in `LivingSituation` on Enrollment.